src/pg/index.js

```diff
@@ -16,7 +16,7 @@ async function find(type, id) {
   return results[0];
 }
 
-async function findAll(type, where = { }, page = 0, perPage = 10) {
+async function findAll(req, type, where = { }, page = 0, perPage = 10) {
```

Comment:
Is the only reason we are passing `req` here to store the connection on it later? Because if so, than passing `req` can be avoided entirely since the session is/should be a global constant.